.github/workflows/spotlight-staging.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           REACT_APP_AUTH_ENABLED: true
           REACT_APP_AUTH_ENV: development
           REACT_APP_API_URL: ${{ secrets.REACT_APP_API_URL }}
-          REACT_APP_ENABLED_TENANTS: US_ND,US_PA
+          REACT_APP_ENABLED_TENANTS: US_ID,US_ND,US_PA,US_TN
```

Comment:
in addition to this you should update the config files in the vault for staging and local development